en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+    ```
+    
+### **Self Registration with External Notification Management**
+
+1. Use the following command to create a user self-registration request. 
+    
+    ``` tab="If the preferred channel is EMAIL" 
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username": "kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/preferredChannel","value": "EMAIL"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "947721584558"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+    ```
+    
+    ``` tab="If the Preferred channel is SMS" 
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username":"kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/preferredChannel","value": "SMS"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "94774747551"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+    ```
+    !!! note
+        for more information, see 
+        [API Requests and Responses](.././learn/extended-self-registration-api-and-account-confirmation-api/#api-requests-and-responses).
+
+2.  Following response will be returned by the API.     
+    
+    ```
+    {
+    "code":"USR-02002",
+    "message":"Successful user self registration. Pending External verification.",
+    "notificationChannel":"EXTERNAL",
+    "confirmationCode":"64928c66-18c2-4092-923b-fb8ce10eeccc"
+    }
+    ```
+   
+    !!! note
+        Note that a confirmation code has been returned by the API. This code needs to be used 
+        to confirm the self registration flow.
+        
+3. Verify the user via an external mechanism.
+
+4. To confirm the verification flow, use the following command.
+
+    ```curl
+    curl -k -v -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H  "Content-Type: application/json" -d '{ "code": "84325529-8aa7-4851-8751-5980a7f2d9f7","verifiedChannel":{"type":"SMS", "claim":"http://wso2.org/claims/mobile"},"properties": []}' "https://localhost:9443/api/identity/user/v1.0/validate-code"
+    ```
+    
+    !!! note
+        Note that you can specify the verified channel in the request. So specify the externally 
+        verified channel, enter channel name and the value claim associated to it.
```

Comment:
Shall we change this sentence 
'Note that you can specify the externally verified channel in the request by entering the channel name and the value claim associated to it'

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549